src/style.css

```diff
@@ -1,3 +1,69 @@
+* {
```

Comment:
Don't use * selector (it impacts performance), better use ``html`` or ``body`` tag.
![image](https://github.com/mate-academy/layout_moyo-header/assets/54860641/790a7dc5-ffd9-48c1-a0d8-94c092e53ff0)